app/models/course.rb

```diff
@@ -11,14 +11,14 @@
 #  discarded_at              :datetime
 #  english                   :integer
 #  id                        :integer          not null, primary key
-#  is_send                   :boolean          default(FALSE)
+#  is_send                   :boolean          default("false")
```

Comment:
Not sure how these annotation updates relate to the commit. Would a separate commit be better?

Reply:
I think the linter keeps catching this or something, I'll remove it from the commits.